bladex/shaft.py

```diff
@@ -0,0 +1,40 @@
+import os
+from OCC.Core.IGESControl import IGESControl_Reader
+from OCC.Core.BRepBuilderAPI import BRepBuilderAPI_MakeSolid, BRepBuilderAPI_Sewing
+import OCC.Core.TopoDS
+
+class Shaft(object):
+    """
+    Bottom-up parametrized shaft construction.
+
+    :param string shaft_path: path of a .iges file with stored shaft information.
+    """
+
+    def __init__(self, shaft_path):
+        self.shaft_path = shaft_path
+
+    def generate_shaft_solid(self):
+        """
+        Generate an assempled solid shaft using the BRepBuilderAPI_MakeSolid 
+        algorithm. 
+        This method requires PythonOCC (7.4.0) to be installed.
```

Comment:
I would remove the version dependency from these docstrings: all the package needs 7.4.0 right?

Reply:
That's fine, thanks.